test/Microsoft.IdentityModel.Validators.Tests/MicrosoftIdentityIssuerValidatorTest.cs

```diff
@@ -595,11 +594,16 @@ public void Validate_FromB2CAuthority_WithTfpIssuer_ThrowsException()
         }
 
         [Theory]
-        [InlineData(true, true)]
-        [InlineData(true, false)]
-        [InlineData(false, false)]
-        [InlineData(false, true)]
-        public void Validate_WithAuthorityUsingConfigurationProvider(bool isV2Token, bool isV2Authority)
+        [InlineData(ProtocolVersion.V1, ProtocolVersion.V1)]
+        [InlineData(ProtocolVersion.V1, ProtocolVersion.V11)]
+        [InlineData(ProtocolVersion.V1, ProtocolVersion.V2)]
+        [InlineData(ProtocolVersion.V11, ProtocolVersion.V1)]
+        [InlineData(ProtocolVersion.V11, ProtocolVersion.V11)]
+        [InlineData(ProtocolVersion.V11, ProtocolVersion.V2)]
+        [InlineData(ProtocolVersion.V2, ProtocolVersion.V1)]
+        [InlineData(ProtocolVersion.V2, ProtocolVersion.V11)]
+        [InlineData(ProtocolVersion.V2, ProtocolVersion.V2)]
```

Comment:
Can this be CombinatorialData instead? Same question for the other two tests

Reply:
I'm not aware of that. How would that look like? Would that be blocking a sign off?

Reply:
Not blocking. Here's the repo:
https://github.com/aarnott/Xunit.Combinatorial

Reply:
I'm not sure I would feel comfortable taking dependency on this.